x/concentrated-liquidity/pool_test.go

```diff
@@ -249,82 +249,6 @@ func (s *KeeperTestSuite) TestCalculateSpotPrice() {
 	s.Require().True(spotPrice.IsNil())
 }
 
-func (s *KeeperTestSuite) TestGetTotalPoolLiquidity() {
```

Comment:
Why is the test removed here? Please add it back

Having a similar test in poolmanager doesn't make this one obsolete

Reply:
re-added